tekton/src/pipelines/gitops/provision-mas-instance.yml.j2

```diff
@@ -255,3 +261,91 @@ spec:
       workspaces:
         - name: configs
           workspace: configs
+{% if provision_manage_addons == true %}
```

Comment:
The condition here should be a `when` condition rather than a templating condition. This would then when condition check is done each time the pipeline runs rather than when the pipeline resource is created. the condition would be something like the following for the kafka and kafka-config task:

```
      when:
      - input: "$(params.mas_app_channel_iot)"
        operator: notin
        values: [""]
```

I think for efs we should just install it as mulitple apps and fvt itself need efs.

Reply:
added the conditional logic to  cli/tekton/src/pipelines/gitops/provision-mas-instance.yml.j2